src/gui/map/rotate_map_dialog.cpp

```diff
@@ -83,14 +83,14 @@ RotateMapDialog::RotateMapDialog(const Map& map, QWidget* parent, Qt::WindowFlag
 	layout->addRow(Util::Headline::create(tr("Options")));
 	
 	adjust_georeferencing_check = new QCheckBox(tr("Adjust georeferencing reference point"));
-	if (map.getGeoreferencing().isValid())
+	if (map.getGeoreferencing().getState() == Georeferencing::Geospatial)
 		adjust_georeferencing_check->setChecked(true);
 	else
 		adjust_georeferencing_check->setEnabled(false);
 	layout->addRow(adjust_georeferencing_check);
 	
 	adjust_declination_check = new QCheckBox(tr("Adjust georeferencing declination"));
-	if (map.getGeoreferencing().isValid())
+	if (map.getGeoreferencing().getState() == Georeferencing::Geospatial)
 		adjust_declination_check->setChecked(true);
```

Comment:
Yes, it could be checked for broken geospatial.

When georeferencing is Local, the meaning of "declination" to Mapper is debatable because Mapper has no use for geographic north. However, grivation _is_ meaningful, and the former version of the code allows the reasonable adjustment of grivation. I would keep that.

Reply:
Hm, IMO grivation is not meaningful with out a valid grid. But declination is: It describes a property of the place with regard to magnetic north and true north.

I see two use cases for rotating maps that are oriented to magnetic north:
a) Updating the declination. Reason: Magnetic north moved.
  Here, the activity should start from editing the declination value in the georeferencing dialog. The user shall not call the rotate-map dialog directly. But he might do so if he doesn't care about semantics/georeferencing. Would he know than that he would need the checkbox to be checked?
b) Fixing the map content. Reason: Map content has an invalid rotation with regard to magnetic north.
  Will the user know that he needs to uncheck the checkbox now?
  If this is the only valid use case, we don't need to deal with declination at all...

Reply:
I agree that when georeferencing is Local, declination is meaningful to the human mapper.

Reply:
Where does the user start? Will the user know to check/uncheck the checkbox? These were some of the issues motivating my experimentation in #1702.   
Your reasoning makes sense, where it concludes that case (b) is the valid use case for directly rotating, and that the dialog does not involve changing the declination. (Aside: "Rotate map" is a faulty description of the effect.)